src/redux/books/bookReducer.js

```diff
@@ -1,13 +1,48 @@
-import * as Actions from './bookAction';
+import { v4 as uuidv4 } from 'uuid';
+import * as actions from '../../action/bookAction';
 
```

Comment:
- [ ] You should assign values for the action types here as per the [ducks pattern](https://github.com/erikras/ducks-modular-redux) indicated in the project's requirement. Kindly make changes here. :pray:

Reply:
- wasn't clear at first what I needed to do, but look into that thank you!

Reply:
Thank you for making the changes and sorry I didn't see your separate `actions` folder which was supposed to be inside the `redux` directory. Anyways your new actions and reducers are far cleaner. :clap: